src/main/fox_versus.c

```diff
@@ -0,0 +1,2018 @@
+
+s32 func_800C0E78(void) {
+    s32 i;
+    u8* D_800D4C80[] = {
+        "FOX",
+        "PEPPY",
+        "SLIPPY",
+        "FALCO",
+    };
+    f32 D_800D4C90[5] = {
+        32.0f, 168.0f, 32.0f, 168.0f, 0.0f,
+    };
+    f32 D_800D4CA4[5] = {
+        20.0f, 20.0f, 200.0f, 200.0f, 0.0f,
+    };
+
+    for (i = 0; i < 4; i++) {
+        if (gControllerPlugged[i] == 1) {
+
+            RCP_SetupDL(&gMasterDisp, 0x53);
+            gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 0, 255);
+            Graphics_DisplaySmallText(D_800D4C90[i], D_800D4CA4[i], 1.0f, 1.0f, D_800D4C80[i]);
+
+            gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 255, 255);
+            func_800869A0(D_800D4C90[i], D_800D4CA4[i] + 9.0f, D_80178838[i], 1.0f, 0, 999);
+        }
+    }
+
+    return 0;
+}
+
+bool func_800C107C(s32 arg0) {
+    D_80178798++;
+    if (D_80178798 == 10) {
+        Audio_PlaySfx(0x49004007U, &D_800C5D28, 4U, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+        D_801787B8++;
+        D_80178798 = 0;
+    }
+
+    if (D_801787B8 < D_80177DB8[arg0]) {
+        return false;
+    } else {
+        return true;
+    }
+}
+
+s32 func_800C1138(s32 max, s32 arg1) {
+    f32 D_800D4CB8[] = { 24.0f, 248.0f, 24.0f, 248.0f };
+    f32 D_800D4CC8[] = { 104.0f, 104.0f, 216.0f, 216.0f };
+    s32 D_800D4CD8[] = { 177, 255, 0, 30 };
+    s32 D_800D4CE8[] = { 242, 30, 179, 30 };
+    s32 D_800D4CF8[] = { 12, 0, 67, 255 };
+    s32 i;
+    s32 j;
+
+    for (i = 0; i < max; i++) {
+        RCP_SetupDL(&gMasterDisp, 0x4D);
+        j = D_80177DD0[arg1][i];
+        gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 255, 255);
+        gDPSetEnvColor(gMasterDisp++, D_800D4CD8[j], D_800D4CE8[j], D_800D4CF8[j], 0);
+        func_800BD720(D_800D4CB8[arg1] + (i * 9.0f), D_800D4CC8[arg1]);
+    }
+
+    return 0;
+}
+
+void func_800C1368(void) {
+    s32 i;
+
+    for (i = 0, D_801787B4 = 0; i < 4; i++) {
+        if (!gPlayerInactive[i]) {
+            D_801787B4++;
+        }
+        D_80178780[i] = 0;
+        D_801787D8[i] = 0.0f;
+        D_801787E8[i] = 0.0f;
+    }
+
+    D_801787CC = 0.0f;
+    D_801787D0 = 0.0f;
+    D_801787AC = 255.0f;
+    D_801787B0 = 0.0f;
+    D_80161A10 = 0.5f;
+    D_80178778 = 0;
+    D_80178790 = 4;
+    D_80178794 = 1;
+    D_8017879C = 0;
+    D_801787A0 = 0;
+    D_801787A4 = 0;
+    D_801787A8 = 99;
+    D_801787B8 = 0;
+    D_80178750 = 0;
+    gBgColor = 0;
+    D_80178830 = 0;
+    D_800D4A90 = 0;
+    D_800D4A98 = 0;
+    D_800D4A9C = D_800D4AA0 = 0;
+    D_80178340 = 0xFF;
+    D_8017835C = 0;
+    D_80178358 = 0;
+
+    for (i = 0; i < 4; i++) {
+        D_80177C30[i] = 0;
+        D_80178808[i] = 0;
+        D_80178810[i] = 0;
+        D_80178820[i] = 0;
+    }
+
+    // clang-format off
+    for (i = 0; i < 4; i++) { D_801787F8[i] = 150; }
+    // clang-format on
+
+    if (gSaveFile.save.data.planet[SAVE_SLOT_VENOM_2].normalClear == 1) {
+        sUnlockLandmaster = 1;
+    } else {
+        sUnlockLandmaster = 0;
```

Comment:
```suggestion
        sUnlockLandmaster = false;
```
and similarly with the others